core/src/main/java/tc/oc/pgm/api/match/Match.java

```diff
@@ -407,6 +407,20 @@ default boolean finish() {
    */
   boolean calculateVictory();
 
+  /**
+   * Get whether friendly fire should be on or off.
+   *
+   * @return True if friendly fire is on.
+   */
+  boolean getFriendlyFire();
+
+  /**
+   * Set an override for friendly fire
+   *
+   * @param allow True to allow friendly fire
+   */
+  void setFriendlyFireOverride(boolean allow);
```

Comment:
```suggestion
  void setFriendlyFireOverride(Boolean allow);
```
true or false to set it to true or false, or null to remove the override